themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
you can use this to apply the styling to text only code. 

```
    ```text
    your .hljs styled   text comes here
    ```
```